src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/BehandlingsoversiktService.kt

```diff
@@ -86,52 +79,25 @@ class BehandlingsoversiktService(
             behandlinger
                 .filter { it.resultat != BehandlingResultat.HENLAGT }
                 .associate { it.id to it.revurderFra }
-        return vedtakRepository
-            .findAllById(behandlinger.map { it.id })
-            .associateBy { it.behandlingId }
-            .mapValues { (behandlingId, vedtak) ->
-                utledVedstaksperiodeForBehandling(vedtak, revurderFraPåBehandlingId[behandlingId])
-            }
+
+        return revurderFraPåBehandlingId.mapValues { (behandlingId, revurderFra) ->
+            slåSammenVedtaksperioderForBehandling(behandlingId, revurderFra)
+        }
     }
 
     /**
-     * Alle vedtaksperioder i beregningsreulatet blir lagret ned i en revurdering.
-     * Dvs hvis jag revurderer fra 15 april, så lagrer vi ned perioder før det og.
-     *
-     * Av den grunnen hentes min(it.fom) og bruker max(minFom, revurderFra),
-     * då revurderFra egentlige gjøres etter minFom på vedtaksperioden
-     *
-     * I tilfelle man kun har opphørt perioder, så vil revurderFra kunne være etter maksTom,
-     * av den grunnen settes tom=max(maksTom, revurderFra)
+     * Slår sammen alle vedtaksperioder som finnes i en behandling slik at oversikten kun viser en periode.
+     * Hvis det er innvilget flere vedtaksperioder med mellomrom i samme behandling, vil disse vises
+     * som en sammenhengende periode, med en fom = første fom-dato og tom = siste tom-dato.
```

Comment:
Dette tror jeg blir en veldig vanlig case med boutgifter. Spesielt med "midlertidig overnatting". Gir det fortsatt mening å slå vedtaksperiodene sammen hvis det er opphold mellom de? For meg virker det litt rart.